core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -179,9 +190,23 @@ class Analyzer(
       // disable it when we explicitly want it to be quiet, instead of
       // enabling on every incoming message.
       reporter.enable()
+      countdown.cancel()
+      countdown = setCountdown()
       allTheThings(req)
   }
 
+  def suspended: Receive = withLabel("suspended") {
+    case req: RpcAnalyserRequest =>
+      // re-start the pc
```

Comment:
this comment serves no purpose. A good rule with code comments is that they should only be used to explain something that is not obvious from reading the code.